compiler/rustc_lint/src/builtin.rs

```diff
@@ -597,7 +596,11 @@ impl<'tcx> LateLintPass<'tcx> for MissingDoc {
         match it.kind {
             hir::ItemKind::Trait(.., trait_item_refs) => {
                 // Issue #11592: traits are always considered exported, even when private.
```

Comment:
This statement about traits looks outdated, and the `private_traits` set is likely unnecessary now, but it's probably a change for a different PR.